src/dali/plugin/input/csv/blockfi.py

```diff
@@ -133,9 +177,98 @@ def load(self) -> List[AbstractTransaction]:
                             crypto_received=Keyword.UNKNOWN.value,
                         )
                     )
-                elif line[self.__TYPE_INDEX] == _WITHDRAWAL_FEE:
+                elif entry_type == _ACH_DEPOSIT:
+                    last_withdrawal_fee = None
+                    result.append(
+                        InTransaction(
+                            plugin=self.__BLOCKFI,
+                            unique_id=Keyword.UNKNOWN.value,
+                            raw_data=raw_data,
+                            timestamp=f"{line[self.__TIMESTAMP_INDEX]} -00:00",
+                            asset=line[self.__CURRENCY_INDEX],
+                            exchange=self.__BLOCKFI,
+                            holder=self.account_holder,
+                            transaction_type="Buy",
+                            spot_price=Keyword.UNKNOWN.value,
+                            crypto_in=line[self.__AMOUNT_INDEX],
+                            fiat_fee="0",
+                            notes="Ach deposit",
+                        )
+                    )
+                elif entry_type == _TRADE:
+                    # Trades will be handled by parsing trade_report_all.csv
+                    # export
+                    continue
+                elif entry_type == _WITHDRAWAL_FEE:
                     last_withdrawal_fee = RP2Decimal(line[self.__AMOUNT_INDEX])
                     last_withdrawal_fee = -last_withdrawal_fee  # type: ignore
                 else:
-                    self.__logger.debug("Unsupported transaction type (skipping): %s", raw_data)
+                    self.__logger.error("Unsupported transaction type (skipping): %s", raw_data)
+
+        if self.__trade_csv_file:
+            result += self.parse_trade_report(self.__trade_csv_file)
+        return result
+
+    def parse_trade_report(self, file_path: str) -> List[AbstractTransaction]:
+        result: List[AbstractTransaction] = []
+
+        with open(file_path, mode="r", encoding="utf-8") as csv_file:
+            lines = reader(csv_file)
+            # skip csv header
+            header = next(lines)
+            self.__logger.debug("Header: %s", header)
+
+            col_idx = {}
+            for (idx, name) in enumerate(header):
+                col_idx[name] = idx
+
+            for line in lines:
+                raw_data: str = self.__DELIMITER.join(line)
+                self.__logger.debug("Transaction: %s", raw_data)
+
+                trans_type = line[col_idx["Type"]]
+                if trans_type != "Trade":
```

Comment:
The code uses full words in identifiers:
```suggestion
                 if transaction_type != "Trade":
```